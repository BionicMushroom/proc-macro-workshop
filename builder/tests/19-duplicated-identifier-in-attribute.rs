use derive_builder::Builder;

#[derive(Builder)]
struct Test {
    #[builder(each = "add_field")]
    field0: Vec<u32>,

    #[builder(each = "add_field")]
    field1: Vec<u32>,
}

fn main() {}
