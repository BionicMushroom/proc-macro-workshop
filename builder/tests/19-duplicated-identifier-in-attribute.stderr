error: identifier `add_field` is duplicated
 --> tests/19-duplicated-identifier-in-attribute.rs:8:22
  |
8 |     #[builder(each = "add_field")]
  |                      ^^^^^^^^^^^
